docs/README_ko.md

```diff
@@ -0,0 +1,79 @@
+<h1 align="">Indonesian Area API (<i>API Wilayah Indonesia</i>)</h1>
+
+<p>
+  <a href="https://nestjs.com"><img alt="NestJS" src="https://img.shields.io/badge/-NestJS-ea2845?style=flat-square&logo=nestjs&logoColor=white" /></a>
+  <a href="https://www.typescriptlang.org"><img alt="TypeScript" src="https://img.shields.io/badge/-TypeScript-007ACC?style=flat-square&logo=typescript&logoColor=white" /></a>
+  <a href="https://www.prisma.io"><img alt="Prisma" src="https://img.shields.io/badge/-Prisma-1B222D?style=flat-square&logo=prisma&logoColor=white" /></a>
+  <a href="https://www.mongodb.com"><img alt="MongoDB" src="https://img.shields.io/badge/-MongoDB-47A248?style=flat-square&logo=mongodb&logoColor=white" /></a>
+  <a href="https://www.postgresql.org"><img alt="PostgreSQL" src="https://img.shields.io/badge/-PostgreSQL-657991?style=flat-square&logo=postgresql&logoColor=white" /></a>
+  <a href="https://www.mysql.com"><img alt="MySQL" src="https://img.shields.io/badge/-MySQL-00688F?style=flat-square&logo=mysql&logoColor=white" /></a>
+</p>
+
+**English** | [हिन्दी](docs/README_hi.md) | [Tagalog](docs/README_tl.md) | [한국어](docs/README_ko.md)
+
+인도네시아의 행정 구역 정보를 제공하는 API입니다. 이 API는 성, 지방, 지구 및 마을 수준의 정보뿐만 아니라 [버전 1.1.0](https://github.com/fityannugroho/idn-area/releases/tag/v1.1.0)부터는 섬 데이터도 제공합니다.
+
+이 API는 [NestJS 프레임워크](https://nestjs.com)로 구축되었으며 TypeScript로 작성되었습니다. 데이터베이스와 상호 작용하기 위해 [Prisma](https://www.prisma.io)를 ORM으로 사용합니다. 이 API는 MySQL, PostgreSQL 및 MongoDB와 같은 다양한 종류의 데이터베이스와 상호 작용할 수 있습니다.
+
+> **⚠️ v3.0.0 로 업데이트 됐습니다. ⚠️**
+>
+> [v3.0.0](https://github.com/fityannugroho/idn-area/releases/tag/v3.0.0) 부터, **Node.js v18** 또는 그 이상의 버전이 요구됩니다.
+
+<h2>목차</h2>
+
+- [시작하기](#getting-started)
+- [데이터](#data)
+- [문서](#documentation)
+- [라이브 데모](#live-demo)
+- [기여](#contributing)
+- [문제 보고](#problem-reporting)
+  - [버그 보고](#reporting-a-bug)
+  - [새로운 기능 요청](#requesting-a-new-feature)
+  - [질문하기](#asking-a-question)
+- [이 프로젝트 지원하기](#support-this-project)
```

Comment:
The links in the table of content (TOC) also need to be translated to make it work. The link is a **snake-case** version of the header **without any symbol**, for example:

| Header | Link |
|--------|--------|
| 이 프로젝트 지원하기 | #이-프로젝트-지원하기 |
| 질문하기 | #질문하기 | 

So that when the reader clicks a link on the TOC, the page will scroll down to the intended section.

Reply:
Also done!